cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1529,6 +1531,36 @@ function Start_Containerd {
   Log-Output "Starting containerd service"
   Start-Service containerd
 }
+
+$PIGZ_ROOT = 'C:\pigz'
+
+# Install Pigz (https://github.com/madler/pigz) into Windows for improved image
+# extraction performance.
+function Install-Pigz {
+  if ("${env:WINDOWS_ENABLE_PIGZ}" -eq "true") {
+    if (-not (Test-Path $PIGZ_ROOT)) {
+      Log-Output "Installing Pigz for Windows"
+      New-Item -Path $PIGZ_ROOT -ItemType Directory
+      MustDownload-File `
+        -Url 'https://storage.googleapis.com/gke-release/winnode/pigz/prod/gke_windows/pigz/release/5/20201104-134221/pigz-2.3.1.zip' `
```

Comment:
1. the path and hash below are hard-coded here, probably better to make it configurable
2. better to have shallower path, 'prod/gke_windows/pigz/release/5/20201104-134221' does not seem to be necessary

Reply:
Should the file be specified in kubeenv or is it sufficient for there to be a constant for the hash and URL. I see both approaches in this file. I'm inclined to follow crictl's lead since pigz will very rarely need updating and there's no strong kubernetes dependency I can think of that needs to update with it.

Reply:
either is fine with me. using kubenv is more flexible, but i'll leave it to you.

Reply:
I decided to follow crictl's lead since it should be updated in lockstep with the CRI.